prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -336,12 +336,13 @@ func findReviewer(ghc githubClient, log *logrus.Entry, useStatusAvailability boo
 		}
 		busy, err := isUserBusy(ghc, candidate)
 		if err != nil {
-			log.Errorf("error checking user availability: %v", err)
+			log.Errorf("error checking user %s availability: %v", candidate, err)
```

Comment:
Let's log this in a structured way (also a good opportunity to log the *error* in a structured way): 

`log.WithField(...).WithError(err).Error("message")`

Same for the second message.